Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/PopupMenuController.java

```diff
@@ -130,24 +139,36 @@ private void addCutMenuItem(Menupopup popup) {
 		Menuitem item = new Menuitem();
 		item.setLabel(Labels.getLabel("common_cut_text"));
 		item.setImage("~./themes/ap/common/img/icons/cut.svg");
-		item.addEventListener(ON_CLICK, event -> getBaseListboxController().cut());
+		if(!popUpOnTree)
+			item.addEventListener(ON_CLICK, event -> getBaseListboxController().cut());
+		else
+			item.addEventListener(ON_CLICK, event -> getNavigationController().cut(selectedFolder));
```

Comment:
We generally go with always having { } around conditional blocks, even in cases like this in which it's not syntactically required.  It also occurs to me that you can probably avoid a little more duplication as follows:
```
item.addEventListener(ON_CLICK, popupOnTree ? event -> getNavigationController().cut(selectedFolder)
                                            : event -> getBaseListboxController().cut());
```